### PR TITLE
Feat: Style SKU Selectors

### PR DESCRIPTION
**Pull request by renatamottam**
## What's the purpose of this pull request?
Styles our two types (for now) of SKU Selectors:
- Size
- Color

## How it works? 
|Prototype|Image|Video|
|-|-|-|
|image|image|https://user-images.githubusercontent.com/11613011/147496628-c4f3e658-3041-49be-9a59-2f92bb616975.mov|

## How to test it?
You can see it working on the `/pattern-library` page

### CODE REVIEW

**Review by lucasfp13 — APPROVED** [review]
Awesome job! 🚀

**Review by ArthurTriis1 — APPROVED** [review]

**Review by saranicoly — APPROVED** [review]
Great job 👏

File: src/components/ui/SkuSelector/SkuSelector.tsx [hunk 1]
```diff
@@ -85,7 +87,7 @@ function SkuSelector({
     <div data-store-sku-selector data-testid={testId} data-variant={variant}>
       {label && (
         <Label data-sku-selector-label>
-          {label}: {selectedSku}
+          {label}: <strong>{selectedSku}</strong>
```
Comment (eduardoformiga) [inline src/components/ui/SkuSelector/SkuSelector.tsx:90]: Unless we are judging that the text between the `<strong>` tag is so important, we should keep it. But if it is just for style purposes, I think we can use `font-weight`?

ref: https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/strong#notas_de_uso

File: src/pages/pattern-library.tsx [hunk 2]
```diff
@@ -88,6 +89,37 @@ function Page() {
...
+                  variant="color"
+                  options={[
+                    { label: 'Yellow', value: '#f1d096' },
+                    { label: 'Pink', value: '#eed0d0' },
+                    { label: 'Green', value: '#b2dbcb' },
+                    { label: 'Blue', value: '#bacbec' },
+                    { label: 'Lilac', value: '#ebdcff', disabled: true },
+                  ]}
+                />
+              </li>
```
Comment (eduardoformiga) [inline src/pages/pattern-library.tsx:119]: I miss the image variant. Not blocking

**Review by eduardoformiga — APPROVED** [review]
The Queen of css 👑.  
Amazing! Left some comments but not blocking.

File: src/components/ui/SkuSelector/SkuSelector.tsx (hunk 1 above)
Comment (renatamottam) [reply src/components/ui/SkuSelector/SkuSelector.tsx:90]: It's more important for colors, for accessibility proposes

File: src/pages/pattern-library.tsx (hunk 2 above)
Comment (renatamottam) [reply src/pages/pattern-library.tsx:119]: It will only be styled after (for the Beta version)

**Pull request merged**